app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -12,8 +12,13 @@
       text: 'Schedule'
     }
   ]
+  publish_time ||= Time.current.tomorrow.change(hour: 9)
```

Comment:
Consider renaming this to `default_publish_time` or `placeholder_time` so it's clear whats being populated in the component.

Reply:
Have now moved this to the controller 👍